packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -272,6 +272,14 @@ export default class extends Component {
                     {host.cpus.cores} ({host.cpus.sockets})
                   </td>
                 </tr>
+                <tr>
+                  <th>{_('hyperThreading')}</th>
+                  <td>
+                    {/\bht\b/.test(host.CPUs.flags)
```

Comment:
You can create a small selector outside the render to avoid running the regexp too many times:
```js
_isHtEnabled = createSelector(
  () => this.props.host.CPUs.flags,
  flags => /\bht\b/.test(flags)
)
```